scss/titles.scss

```diff
@@ -18,4 +18,20 @@
     width: 70px;
     z-index: -1;
   } 
-}
+}  
+
+@media only screen and (max-width: 1024px) {
+  .header-title {
+    display: none;
+  }
+
+  .page-title {
```

Comment:
`.page-title::before { ...} `
Do you really need to override all those attributes?

Reply:
Not really. Just did it to achieve the card's layout image.